docker-compose.yml

```diff
@@ -1,6 +1,21 @@
 version: '3'
 services:
+  test:
+    build: .
+    command: 'npm run test:docker:ci'
+    depends_on:
+      - rediscluster
+      - rabbitmq
+    volumes:
+      - .:/usr/windbreaker
+      - /usr/windbreaker/node_modules
```

Comment:
Another neat trick. We mount the the service utils into the container. Specifying the `node_modules` folder like this forces docker to remount whatever was originally in that location (the modules installed from the build). This let's us make quick changes without having to do rebuilds.